src/TokensTracker/Trackers/NEP-11/Nep11Tracker.cs

```diff
@@ -0,0 +1,287 @@
+using Neo.IO.Json;
+using Neo.Ledger;
+using Neo.Network.P2P.Payloads;
+using Neo.Persistence;
+using Neo.SmartContract;
+using Neo.SmartContract.Native;
+using Neo.VM;
+using Neo.VM.Types;
+using Neo.Wallets;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using Array = Neo.VM.Types.Array;
+
+namespace Neo.Plugins.Trackers.NEP_11
+{
+    class Nep11Tracker : TrackerBase
+    {
+        private const byte Nep11BalancePrefix = 0xf8;
+        private const byte Nep11TransferSentPrefix = 0xf9;
+        private const byte Nep11TransferReceivedPrefix = 0xfa;
+        private uint _currentHeight;
+        private Block _currentBlock;
+        private readonly HashSet<string> _properties = new()
+        {
+            "name",
+            "description",
+            "image",
+            "tokenURI"
+        };
+
+
+        public Nep11Tracker(IStore db, uint maxResult, bool shouldRecordHistory, NeoSystem system) : base(db, maxResult, shouldRecordHistory, system)
+        {
+        }
+
+        public override void OnPersist(NeoSystem system, Block block, DataCache snapshot, IReadOnlyList<Blockchain.ApplicationExecuted> applicationExecutedList)
+        {
+            _currentBlock = block;
+            _currentHeight = block.Index;
+            uint nep11TransferIndex = 0;
+            var transfers = new List<TransferRecord>();
+            foreach (Blockchain.ApplicationExecuted appExecuted in applicationExecutedList)
+            {
+                // Executions that fault won't modify storage, so we can skip them.
+                if (appExecuted.VMState.HasFlag(VMState.FAULT)) continue;
+                foreach (var notifyEventArgs in appExecuted.Notifications)
+                {
+                    if (notifyEventArgs.EventName != "Transfer" || !(notifyEventArgs?.State is Array stateItems) ||
+                        stateItems.Count == 0)
+                        continue;
+                    var contract = NativeContract.ContractManagement.GetContract(snapshot, notifyEventArgs.ScriptHash);
+                    if (contract?.Manifest.SupportedStandards.Contains("NEP-11") == true)
+                    {
+                        HandleNotificationNep11(notifyEventArgs.ScriptContainer, notifyEventArgs.ScriptHash, stateItems,
```

Comment:
It could fault and deny the service

Reply:
Yes. If it run into fault, we should let it down.